get_deps.py

```diff
@@ -30,11 +30,24 @@ def main():
 		if dll_name.startswith('???'):
 			print('Unknown DLL?')
 			continue
-		if dll_path.lower().startswith('/c/windows') or dll_path.lower().startswith('c:/windows'):
+		try:
+			# ldd on msys2 gives Unix-style paths, but Python wants them Windows-style
+			# Try cygpath to convert them
+			dll_path = subprocess.check_output(['cygpath', '-w', dll_path]).strip()
+		except:
+			# cygpath doesn't exist or failed. Carry on and hope for the best

```

Comment:
I'm not a fan of swallowing exceptions without so much as an error message. Please either (a) `exit(1)` due to `cygpath` being unavailable (it should be in msys2 environment!) or (b) at least print an error message or something.